src/MockeryModule/Test.php

```diff
@@ -16,7 +16,7 @@
  */
 class Test extends BaseTest
 {
-    protected function tearDown()
```

Comment:
Please change parent class to `\Codeception\Test\Unit`,   
rename this method to `_tearDown`, 
remove result typehint,
and raise the lowest supported version of `codeception/codeception` to 2.4.0